src/app/auth/google/google.component.html

```diff
@@ -31,9 +31,15 @@
       </div>
     </div>
     <div class="d-flex pt-15px justify-content-end">
-      <button (click)="finishFlow(true)" class="btn btn-primary font-weight-bold fs-15px sign-up-btn">
+      <button (click)="finishFlow(true)" class="btn btn-primary font-weight-bold fs-15px sign-up-btn" *ngIf="!environment.jumioSupported">
         Continue to {{ globalVars.hostname }}
       </button>
+      <button (click)="startJumio()" class="btn btn-primary font-weight-bold fs-15px sign-up-btn" *ngIf="environment.jumioSupported">
+        Get Free $CLOUT
```

Comment:
@diamondhands0 Is this the text we want for this button?

Reply:
I think he probably wants this:
```suggestion
        Get  <b>{{ globalVars.nanosToUSD(globalVars.jumioBitCloutNanos, 0)}} Free</b>
```